agent/util-scripts/pbench-move-results

```diff
@@ -72,19 +80,39 @@ while true; do
     esac
 done
 
-if [[ ! -d "${pbench_run}" ]]; then
+# Move into pbench run collection directory
+cd ${pbench_run} >/dev/null
+if [[ ${?} -ne 0 ]]; then
     error_log "ERROR: pbench local results directory does not exist: ${pbench_run}"
     exit 1
 fi
 
-if [[ ! -f "${pbench_bin}/id_rsa" ]]; then
-    error_log "ERROR: ${pbench_bin}/id_rsa required for moving results to archive host"
+tmp=${pbench_tmp}/${script_name}.${$}
+trap "rm -rf ${tmp}" EXIT INT QUIT
```

Comment:
Also trap SIGTERM.  Or do you want the directory to stay around if someone manually runs `kill`?